.circleci/config.yml

```diff
@@ -295,3 +311,7 @@ workflows:
             branches:
               only:
                 - master
+  rebase-and-unit-test:
+    when: << pipeline.parameters.run_rebase_and_unit_test >>
```

Comment:
The use of this variable warrants a comment this flow is a bit complex.

Reply:
@roopakv And I discussed this and decided on running the rebase job in the merchant branch itself rather than from the PR branch

Reply:
Agreed, still a comment to explain that this is triggered manually from a PR is worth explaining in a comment. 
It is abnormal for a Circle Config to relate to a commit in another branch